modules/local/gzrt/main.nf

```diff
@@ -0,0 +1,51 @@
+process GZRT {
+    tag "$meta.id"
+    label 'process_low'
+    container 'docker.io/mazzalab/fastqrepair_nf_env:1.0.1'
```

Comment:
Is there a possibility to use a seqera container and/or biocontainers? or submit to bioconda?

Reply:
refer to the answer to the same question in "bbmaprepair"